roles/fio-distributed/templates/job.fio.rand

```diff
@@ -10,7 +10,6 @@ time_based=1
 runtime={{fiod.runtime}}
 clocksource=clock_gettime
 ramp_time={{fiod.ramp_time}}
-stonewall
```

Comment:
Why are we removing stonewall from the jobfiles?

Side note: This only needs to be defined once in the `[global]` section; it is redundant to repeat it in the individual job sections.